introduction.md

```diff
@@ -0,0 +1,3 @@
+Make a local clone of your fork of the "dojo_rules" repository. Make sure you don't fork the deadlyvipers version of the repo – clone your fork of it.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.